src/app/app-routing.module.ts

```diff
@@ -8,13 +9,17 @@ const routes: Routes = [
   },
   {
     path: 'auth',
```

Comment:
add this also to routes constant file

Reply:
Then i cant add / there right?
because no slash here

Reply:
the issue i faced by that is path cannot start with a slash error in app-routing.module.ts